src/glob.zig

```diff
@@ -1006,10 +1006,10 @@ pub fn GlobWalker_(
         }
 
         inline fn startsWithDot(filepath: []const u8) bool {
-            if (comptime !isWindows) {
-                return filepath[0] == '.';
+            if (comptime isWindows) {
+                return filepath.len > 1 and filepath[1] == '.';
```

Comment:
added this so the tests on glob dont panic but not familiar with glob todo a full fix, this PR is intended to fix which